packages/core/src/modules/dids/DidsApi.ts

```diff
@@ -99,4 +104,8 @@ export class DidsApi {
   public getCreatedDids({ method }: { method?: string } = {}) {
     return this.didRepository.getCreatedDids(this.agentContext, { method })
   }
+
+  public createV2DID(options: CreateDIDParams): Promise<DidCreateResult> {
+    return this.didService.createDID(this.agentContext, options)
+  }
```

Comment:
I agree. This is just a temporary method to v2 compatible dids. The current DID's creation logic is tide bind to the routing keys / service, logic. 
In order to prevent breaking existing functionality we just created an extra functionality needed for general v2 demonstration. In the future, we will find ways how to properly integrate it with  DidRegistrar.

Reply:
I'm fine with breaking the interface of existing logic, as long as the existing functionality doesn't break. So feel free to make breaking changes. Rather have breaking changes in the API (we're still pre-1.0, that is expected) than adding hacks around the existing API. If something's missing we should change it rather than work around it.